extensions-contrib/moving-average-query/README.md

```diff
@@ -0,0 +1,35 @@
+<!--
+  ~ Licensed to the Apache Software Foundation (ASF) under one
+  ~ or more contributor license agreements.  See the NOTICE file
+  ~ distributed with this work for additional information
+  ~ regarding copyright ownership.  The ASF licenses this file
+  ~ to you under the Apache License, Version 2.0 (the
+  ~ "License"); you may not use this file except in compliance
+  ~ with the License.  You may obtain a copy of the License at
+  ~
+  ~   http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing,
+  ~ software distributed under the License is distributed on an
+  ~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  ~ KIND, either express or implied.  See the License for the
+  ~ specific language governing permissions and limitations
+  ~ under the License.
+  -->
+
+druid-moving-average-query
+=============
+
+Overview
+=============
+**Moving Average Query** is an extension which provides support for [Moving Average](https://en.wikipedia.org/wiki/Moving_average) and other Aggregate [Window Functions](https://en.wikibooks.org/wiki/Structured_Query_Language/Window_functions) in Druid queries.
+
+Documentation 
+=============
+See the druid.io website or under [Druid Github Repo](https://github.com/apache/incubator-druid/tree/master/docs/content/development/extensions-contrib/moving-average-query.md).
+
+Thanks
+===========
+
+This module was created by [Will Lauer](https://github.com/will-lauer) and [Mangesh Pardeshi](https://github.com/mangesh-pardeshi), with additional contributions by: [Deepak Babu](https://github.com/deepakb91), [Asif Mansoor](https://github.com/asifmansoora), [Ke Wang](https://github.com/kenuiuc), [Eyal Yurman](https://github.com/yurmix).
```

Comment:
@jihoonson Do you prefer that contributors give credit to the authors by listing them in a separate AUTHORS.txt file or in a thank you section in the README? Either way, the copyrights for the project belong to ASF and the contributions are licensed under the terms of the CLA. But it's real people who write this code and we'd like to say thanks somewhere.

Reply:
Generally we avoid crediting individual people in documentation (including javadoc `@author` tags). It's not a slight to them, but it is because we don't want to be in the business of appearing to say that some contributors are worthy of special thanks in documentation and some aren't.

We _do_ typically acknowledgement _all_ contributors in release notes, though, no matter how minor. And we do also typically acknowledge specific people that are behind whichever changes the release manager writes up as the headline changes. Check the credits for 0.13.0 for example: https://github.com/apache/incubator-druid/releases/tag/druid-0.13.0-incubating. The authors of this contribution definitely merit a mention there.

Btw, IMHO if the original authors have published a paper or blog post describing the background of this work - how it came to be, design considerations, and such - then linking to that from the documentation would be totally fine. It doesn't cause the same problem I mentioned earlier, since it's not a credit, it's a link to additional information.

Reply:
This makes total sense. It's just I'm not the sole author of the entire module and I don't feel like implicitly claiming so.
We have a paper and I'll look into publishing it. I'd also suggest when it's being released to manually add a couple of authors in the release notes if possible.
Thank you.